firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/CrashlyticsRemoteConfigListener.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.crashlytics.internal;
+
+import androidx.annotation.NonNull;
+import com.google.firebase.crashlytics.internal.metadata.RolloutAssignment;
+import com.google.firebase.crashlytics.internal.metadata.UserMetadata;
+import com.google.firebase.remoteconfig.interop.rollouts.RolloutsState;
+import com.google.firebase.remoteconfig.interop.rollouts.RolloutsStateSubscriber;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+public class CrashlyticsRemoteConfigListener implements RolloutsStateSubscriber {
+  private final UserMetadata userMetadata;
+
+  CrashlyticsRemoteConfigListener(UserMetadata userMetadata) {
+    this.userMetadata = userMetadata;
+  }
+
+  @Override
+  public void onRolloutsStateChanged(@NonNull RolloutsState rolloutsState) {
+    Set<com.google.firebase.remoteconfig.interop.rollouts.RolloutAssignment> rolloutAssignmentList =
+        rolloutsState.getRolloutAssignments();
+
+    List<com.google.firebase.crashlytics.internal.metadata.RolloutAssignment>
```

Comment:
I think you can just use the unqualified `RolloutAssignment` here like below

Reply:
One `RolloutAssignment` is from interop and another is Crashlytics class. Unfortunately in Java you cannot import same class name from different source

Reply:
Right right. I _think_ this is the one imported from Crashlytics (and referenced below without the full package)?

Reply:
Indeed!